2019-08-12 14:39:22.556 [INF] [app.go:139] services[serviceA]
2019-08-12 14:39:22.622 [DBG] [app.go:76] App.init, appID: 210
2019-08-12 14:39:22.623 [INF] [serviceA.go:24] ServiceA OnInit
2019-08-12 14:39:22.707 [DBG] [serviceProc.go:149] set verify msgid, err: 0
2019-08-12 14:39:22.707 [DBG] [serviceProc.go:153] outer addr: 127.0.0.1:17000
2019-08-12 14:39:22.707 [DBG] [localService.go:70] service check : serviceA
2019-08-12 14:39:22.708 [DBG] [localService.go:100] service check success: serviceA
2019-08-12 14:39:22.708 [DBG] [localServiceMgr.go:50] App InitLocalService, name: serviceA
2019-08-12 14:39:22.809 [DBG] [localService.go:185] run service , serviceName: serviceA, serviceType: 2, ServerID: 211 tickMS: 2000
2019-08-12 14:39:25.918 [DBG] [serviceProc.go:67] MsgProc_LoginReq, uid: 1
2019-08-12 14:39:35.467 [DBG] [dbservice.go:52] connect failed,no reachable servers
2019-08-12 14:39:35.467 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-08-12 14:39:35.467 [INF] [Entity_Props.go:171] select props: map[coin:1 exp:1 level:1 rating:1]props return: map[]
2019-08-12 14:39:35.467 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 2, serviceID: 211
2019-08-12 14:39:35.469 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 1, serviceID: 211
2019-08-12 14:39:35.471 [DBG] [ServiceAUser.go:20] ServiceAUser.OnUserInit, dbid: 1 isnew: true
2019-08-12 14:39:35.471 [INF] [serviceA.go:52] ServiceA OnLoginHandler, msg.UID = 1
2019-08-12 14:39:35.471 [DBG] [ServiceAUserRpcProc.go:14] RPC_Hello, name: hello world, id: 1
2019-08-12 14:39:35.471 [DBG] [ServiceAUserRpcProc.go:16] err: = <nil>
2019-08-12 14:40:03.308 [ERR] [session.go:246] recvLoop error: read tcp 127.0.0.1:17000->127.0.0.1:54151: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54151, sessionid: 1
2019-08-12 14:40:03.308 [ERR] [session.go:254] read and handle message error: read tcp 127.0.0.1:17000->127.0.0.1:54151: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54151, sessionid: 1
2019-08-12 14:40:03.308 [INF] [serviceProc.go:105] Proc_ServiceA OnClosed: 1 127.0.0.1:54151
2019-08-12 14:40:03.309 [DBG] [GateUserBase.go:165] lobby user OnClosed, uid: 1
2019-08-12 14:42:38.951 [INF] [app.go:139] services[serviceA]
2019-08-12 14:42:38.966 [DBG] [app.go:76] App.init, appID: 212
2019-08-12 14:42:39.005 [INF] [serviceA.go:24] ServiceA OnInit
2019-08-12 14:42:39.107 [DBG] [serviceProc.go:149] set verify msgid, err: 0
2019-08-12 14:42:39.107 [DBG] [serviceProc.go:153] outer addr: 127.0.0.1:17000
2019-08-12 14:42:39.107 [DBG] [localService.go:70] service check : serviceA
2019-08-12 14:42:39.107 [DBG] [localService.go:100] service check success: serviceA
2019-08-12 14:42:39.107 [DBG] [localServiceMgr.go:50] App InitLocalService, name: serviceA
2019-08-12 14:42:39.204 [DBG] [localService.go:185] run service , serviceName: serviceA, serviceType: 2, ServerID: 213 tickMS: 2000
2019-08-12 14:42:42.439 [DBG] [serviceProc.go:67] MsgProc_LoginReq, uid: 1
2019-08-12 14:42:51.969 [DBG] [dbservice.go:52] connect failed,no reachable servers
2019-08-12 14:42:51.969 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-08-12 14:42:51.969 [INF] [Entity_Props.go:171] select props: map[coin:1 exp:1 level:1 rating:1]props return: map[]
2019-08-12 14:42:51.969 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 2, serviceID: 213
2019-08-12 14:42:51.971 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 1, serviceID: 213
2019-08-12 14:42:51.972 [DBG] [ServiceAUser.go:20] ServiceAUser.OnUserInit, dbid: 1 isnew: true
2019-08-12 14:42:51.972 [INF] [serviceA.go:52] ServiceA OnLoginHandler, msg.UID = 1
2019-08-12 14:42:51.973 [DBG] [ServiceAUserRpcProc.go:14] RPC_Hello, name: hello world, id: 1
2019-08-12 14:42:51.973 [DBG] [ServiceAUserRpcProc.go:16] err: = <nil>
2019-08-12 14:53:20.692 [ERR] [session.go:246] recvLoop error: read tcp 127.0.0.1:17000->127.0.0.1:54211: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54211, sessionid: 1
2019-08-12 14:53:20.693 [ERR] [session.go:254] read and handle message error: read tcp 127.0.0.1:17000->127.0.0.1:54211: wsarecv: An existing connection was forcibly closed by the remote host., remoteAddr: 127.0.0.1:54211, sessionid: 1
2019-08-12 14:53:20.693 [INF] [serviceProc.go:105] Proc_ServiceA OnClosed: 1 127.0.0.1:54211
2019-08-12 14:53:20.693 [DBG] [GateUserBase.go:165] lobby user OnClosed, uid: 1
2019-08-12 14:54:48.348 [INF] [app.go:139] services[serviceA]
2019-08-12 14:54:48.356 [DBG] [app.go:76] App.init, appID: 214
2019-08-12 14:54:48.399 [INF] [serviceA.go:24] ServiceA OnInit
2019-08-12 14:54:48.483 [DBG] [serviceProc.go:149] set verify msgid, err: 0
2019-08-12 14:54:48.483 [DBG] [serviceProc.go:153] outer addr: 127.0.0.1:17000
2019-08-12 14:54:48.483 [DBG] [localService.go:70] service check : serviceA
2019-08-12 14:54:48.484 [DBG] [localService.go:100] service check success: serviceA
2019-08-12 14:54:48.484 [DBG] [localServiceMgr.go:50] App InitLocalService, name: serviceA
2019-08-12 14:54:48.584 [DBG] [localService.go:185] run service , serviceName: serviceA, serviceType: 2, ServerID: 215 tickMS: 2000
2019-08-12 14:56:02.12 [INF] [app.go:139] services[serviceA]
2019-08-12 14:56:02.129 [DBG] [app.go:76] App.init, appID: 216
2019-08-12 14:56:02.171 [INF] [serviceA.go:24] ServiceA OnInit
2019-08-12 14:56:02.254 [DBG] [serviceProc.go:149] set verify msgid, err: 0
2019-08-12 14:56:02.254 [DBG] [serviceProc.go:153] outer addr: 127.0.0.1:17000
2019-08-12 14:56:02.254 [DBG] [localService.go:70] service check : serviceA
2019-08-12 14:56:02.255 [DBG] [localService.go:100] service check success: serviceA
2019-08-12 14:56:02.255 [DBG] [localServiceMgr.go:50] App InitLocalService, name: serviceA
2019-08-12 14:56:02.356 [DBG] [localService.go:185] run service , serviceName: serviceA, serviceType: 2, ServerID: 217 tickMS: 2000
2019-08-12 14:56:05.257 [DBG] [serviceProc.go:67] MsgProc_LoginReq, uid: 1
2019-08-12 14:56:14.778 [DBG] [dbservice.go:52] connect failed,no reachable servers
2019-08-12 14:56:14.778 [ERR] [mongodboperate.go:89] session nil, not connect or auth
2019-08-12 14:56:14.778 [INF] [Entity_Props.go:171] select props: map[coin:1 exp:1 level:1 rating:1]props return: map[]
2019-08-12 14:56:14.778 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 2, serviceID: 217
2019-08-12 14:56:14.779 [DBG] [entitySrvUtil.go:64] Reg service key: entitysrvinfo:1, serviceType: 1, serviceID: 217
2019-08-12 14:56:14.78 [DBG] [ServiceAUser.go:20] ServiceAUser.OnUserInit, dbid: 1 isnew: true
2019-08-12 14:56:14.78 [INF] [serviceA.go:52] ServiceA OnLoginHandler, msg.UID = 1
2019-08-12 14:56:14.78 [DBG] [ServiceAUserRpcProc.go:14] RPC_Hello, name: hello world, id: 1
2019-08-12 14:56:14.78 [DBG] [ServiceAUserRpcProc.go:16] err: = <nil>
